libraries/bsg_manycore.h

```diff
@@ -171,7 +171,7 @@ extern "C" {
          * @param[in] timeout A timeout counter. Unused - set to -1 to wait forever.
          * @return HB_MC_SUCCESS on success. Otherwise an error code defined in bsg_manycore_errno.h.
          */
-        __attribute__((warn_unused_result, deprecated))
```

Comment:
just out of curiosity, why was this deprecated in the first place?

Reply:
To summarize on slack - I think this was deprecated a long time ago. We don't want users to call it, but we still call it in our tests so that we can "finish" the tests.

Our long term goal (as demonstrated in #665) is to replace a call to this with a function for explicit finish/fail.